cli/src/commands/init.ts

```diff
@@ -4,26 +4,34 @@ import { Options } from "yargs-parser";
 import { anchorBinUrlMap, downloadFileIfNotExists } from "../utils/downloadBin";
 
 const path = require("path");
-export const command: string = "init";
+export const command: string = "init [name]";
```

Comment:
Gosh, we really should change to some CLI library which doesn't require writing such stuff (and in general - help promts) by hand.

Not in this PR, but for sure at some point.